src/helpers/OTSessionHelper.js

```diff
@@ -49,13 +50,26 @@ const sanitizeSignalData = (signal) => {
     };
   }
   return {
-    type: signal.type ? isString(signal.type) : '',
-    data: signal.data ? isString(signal.data) : '',
+    type: signal.type ? validateString(signal.type) : '',
+    data: signal.data ? validateString(signal.data) : '',
     errorHandler: typeof signal.errorHandler !== 'function' ? handleSignalError : signal.errorHandler,
   };
 };
 
+const sanitizeCredentials = (credentials) => {
+  const _credentials = {};
+  each(credentials, (value, key) => {
+    if(!isString(value) || isEmpty(value) || isNull(value)) {
```

Comment:
can we reuse `validateString` here?

Reply:
`validateString` just check's if it's a string and if it is, it returns the value whereas these three simply check if the value is a string, is empty or is null.